pulp_ansible/tests/functional/api/collection/test_signatures.py

```diff
@@ -0,0 +1,199 @@
+"""Tests functionality around Collection-Version Signatures."""
+from pulp_smash.pulp3.bindings import delete_orphans, monitor_task
+from pulp_ansible.tests.functional.utils import (
+    create_signing_service,
+    delete_signing_service,
+    gen_repo,
+    gen_ansible_remote,
+    gen_distribution,
+    get_content,
+    SyncHelpersMixin,
+    TestCaseUsingBindings,
+    skip_if,
+)
+from pulp_ansible.tests.functional.constants import TEST_COLLECTION_CONFIGS
+from orionutils.generator import build_collection
+from pulpcore.client.pulp_ansible import AnsibleCollectionsApi, ContentCollectionSignaturesApi
+from pulp_ansible.tests.functional.utils import set_up_module as setUpModule  # noqa:F401
+
+
+class CRUDCollectionVersionSignatures(TestCaseUsingBindings, SyncHelpersMixin):
```

Comment:
I'm hoping these can be switched to the pytest tests (some other time). We can't afford to take the time to change this prior to merging.